testing/e2e/README.md

```diff
@@ -0,0 +1,35 @@
+# e2e Smoke Tests
```

Comment:
@SB159 can we consolidate all testing scripts in one main script? this will be easier to integrate with prow.